Plugins/IngameDebugConsole/Scripts/Commands/PlayerPrefsCommands.cs

```diff
@@ -0,0 +1,38 @@
+﻿using UnityEngine;
+
+namespace IngameDebugConsole.Commands
+{
+	public class PlayerPrefsCommands
+	{
+		[ConsoleMethod( "pp.get",
+			"Get the value of a PlayerPrefs field",
```

Comment:
Please use present tense to be consistent with _help_ and _sysinfo_. Instead of _Get_, prefer _Returns_